src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+  - /docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability
+  - /docs/agents/manage-apm-agents/installation/install-agent
+  - /docs/new-relic-one/use-new-relic-one/cross-product-functions/install-configure/install-new-relic
+  - /docs
+  - /docs/using-new-relic/cross-product-functions/install-configure/install-new-relic
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
+freshnessValidatedDate: 2024-03-dd
+---
+
+import NRAlertsMain from 'images/alerts_screenshot-crop_main.webp'
+
+import NRApmMain from 'images/apm_screenshot-crop_main.webp'
+
+import NRBrowserMain from 'images/browser_screenshot-crop_main.webp'
+
+import NRDashboardsMain from 'images/dashboards_screenshot-crop_main.webp'
+
+import NRErrosInboxMain from 'images/errors-inbox_screenshot-crop_main.webp'
+
+import NRIastMain from 'images/iast_screenshot-crop_main.webp'
+
+import NRInfraMain from 'images/infrastructure_screenshot-crop_main.webp'
+
+import NRLambdaMain from 'images/lambda_screenshot-crop_main.webp'
+
+import NRLiveArchivesMain from 'images/live-archives_screenshot-crop_main.webp'
+
+import NRLogsMain from 'images/logs_screenshot-crop_main.webp'
+
+import NRK8sMain from 'images/kubernetes_screenshot-crop_main.webp'
+
+import NRMobileMain from 'images/mobile_screenshot-crop_main.webp'
+
+import NRNetworkMain from 'images/network_screenshot-crop_main.webp'
+
+import NROpenTelemetryMain from 'images/opentelemetry_screenshot-crop_main.webp'
+
+import NRServiceLevelsMain from 'images/slm_screenshot-crop_main.webp'
+
+import NRSyntheticMain from 'images/synthetic_screenshot-crop_main.webp'
+
+import NRVulnManagementMain from 'images/vulnerability-management_screenshot-crop_main.webp'
+
+
+Welcome to the New Relic observability platform! Our platform gives you deep visibility into your systems. It helps you solve problems you may not have been aware of, regardless of the technology you use. You can see how New Relic works [here](#how-it-works) or [get started](#get-started) now if you want to jump right in.
+
+New Relic provides many ways to improve visibility. They let you monitor, troubleshoot, and improve digital performance. Here's a brief overview of our most popular capabilities and features.
+
+
+# Alerts [#alerts]
+<SideBySide>
+  <Side>
+Alerts help you identify issues that look unusual in your applications and code. New Relic includes some <InlinePopover type="alerts" /> when you install it, and you can also create your alerts. You receive alerts through tools like PagerDuty, ServiceNow, Jira, Slack, and more. Let us help you know what's critical, manage the suppressed noise, and mitigate alert fatigue. 
+  
+Learn more about alerts [here](/docs/alerts-applied-intelligence/overview/).
+
+  </Side>
+  <Side>
+    <img
+      title="New Relic alerts"
+      alt="New Relic alerts"
+      src={NRAlertsMain}
+    />
+  </Side>
+  </SideBySide>
+
+# APM [#apm]
```

Comment:
I think the title should be "Application monitoring", then we can use APM in the blurb

Reply:
Done!